components/popover/stories/popover.test.js

```diff
@@ -116,11 +120,38 @@ export const PopoverGroup = Variants({
 				testHeading: `Position: ${position}`,
 				wrapperStyles,
 				position,
-				withTestContainer: true,
 				popoverHeight,
 				popoverWidth,
 			};
 		}),
+		{
+			testHeading: "Dialog style content",
```

Comment:
Want to add a `Nested` test?

Reply:
It looks liked Nested is already tested through the separate story in Chromatic. I think that makes sense since it's a special case that's manually positioned and not being tested with all the test case combinations.